.github/workflows/test.yml

```diff
@@ -0,0 +1,11 @@
+name: Test
+on: workflow_dispatch
+jobs:
+  formatter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./formatter
+        with:
+          filetypes: .cpp,.hpp
+          working-directory: ./test
```

Comment:
Maybe we could create another file and add it to the ignored files list to make sure that feature works as well.

Reply:
Done it - you check the logs on my forks actions, you can see that the ignore files works nicely now 😄